docs/sources/configuration/_index.md

```diff
@@ -3939,6 +3943,25 @@ embedded_cache:
   # CLI flag: -<prefix>.embedded-cache.ttl
   [ttl: <duration> | default = 1h]
 
+lru_cache:
```

Comment:
Does LRU stand for anything?  Would it help people understand if it was defined somewhere?

Reply:
Yep, it stands for last-recently used. I'll add a note about that.